LayoutTests/imported/w3c/web-platform-tests/css/css-page/parsing/page-rules-001-expected.txt

```diff
@@ -0,0 +1,10 @@
+
+PASS page-rules-001
+PASS @page , { } should be an invalid rule
+FAIL @page { } should be a valid rule assert_equals: serialization should be canonical expected "@page { }" but got "@page  { }"
```

Comment:
This got regressed because of `extra` space.

Reply:
Can we fix this please? No reason we have to land it broken.

Reply:
Yes - I am looking in background and will definitely try to not ship any regression.

Reply:
No regression anymore. :-)